### PR TITLE
Dependency update, fix documentation, address chefzero url scheme, fixes #15

### LINKED ISSUE #15: Test instructions are invalid, tests missing
It looks like the tests have been taken wholesale from vagrant and not modified to work with lxc.  There are no lxc tests at all.

### PR DESCRIPTION
**Pull request by ranjib**
Fixes https://github.com/chef/chef-provisioning-lxc/issues/15

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,36 +1,32 @@
 GIT
```
Comment (tyler-ball) [inline Gemfile.lock]: I don't even think we should have `Gemfile.lock` checked in - can you just delete this @ranjib?

File: chef-provisioning-lxc.gemspec [hunk 2]
```diff
@@ -10,11 +10,9 @@ Gem::Specification.new do |s|
   s.description = s.summary
   s.author = 'Ranjib Dey'
   s.email = 'dey.ranjib@gmail.com'
-  s.homepage = 'https://github.com/opscode/chef-provisioning-lxc'
+  s.homepage = 'https://github.com/chef/chef-provisioning-lxc'
 
-  s.add_dependency 'chef', '>= 12.0.3'
   s.add_dependency 'chef-provisioning'
```
Comment (tyler-ball) [inline chef-provisioning-lxc.gemspec]: In all the other projects we have `~> 1.0` if they work with 1.0 - can you update that here?

**Comment by ranjib** [comment]
@tyler-ball done.

**Comment by tyler-ball** [comment]
@ranjib thanks for the contribution!

**Pull request merged**